src/Command/RunCommand.php

```diff
@@ -41,10 +48,10 @@ protected function configure(): void
             'Define a specific tool which should be run'
         );
         $this->addOption(
-            'keep-going',
-            'k',
+            'fast-finish',
+            'f',
```

Comment:
I don't like the shortcut here, as it's quite often used as `force` options. I suggest using `ff` instead.